bitnami/kubernetes-event-exporter/Chart.yaml

```
@@ -26,4 +26,4 @@ name: kubernetes-event-exporter
 sources:
   - https://github.com/bitnami/bitnami-docker-kubernetes-event-exporter
   - https://github.com/opsgenie/kubernetes-event-exporter
-version: 1.1.14
+version: 1.1.15
```

For changes in the README it's not needed to bump the chart version
```suggestion
version: 1.1.14
```